### ISSUE TITLE
Order for under/over in UEB

### ISSUE THREAD

**Issue by NSoiffer** [NSoiffer/MathCAT#226]
In GTM 7.9, there is a sum. The part below the sum is written first and the upper limit is written second.  [aphtech.org](https://uebmath.aphtech.org/lesson10.2) uses the same ordering for integrals.

In GTM 16.5, they have a chemistry example with something above and below an arrow. These are written with what is above first and what is below second.

I suspect both are legal.

@rob-aph: is there some guidance as to which should come first and which should come second? For example, "if the base is a large operator (sum, integral, intersection, ...), use the order under followed by over, otherwise write the upper part first followed by the lower part".

**Comment by rob-aph** [comment]
This was beyond my knowledge, so I asked one of our braille experts. Her response follows:

It is my understanding that the Chemical equation in GTM 16.5 is a notable exception to the general rule of order of modifiers in UEB. I suspect that in that particular equation, the material under the arrow “Haber process” is a label for the arrow, which has the Chemically relevant information “H₂” above it. So, an exception is made to the general rule for ordering modifiers so that the label can follow all the parts of the item it’s labeling.
In other words, it is pretty much always best to braille the modifier below first.

**Comment by NSoiffer** [comment]
@rob-aph: can you ask your expert what they would do in other situations where there is an arrow with information above and below the arrow. Would they braille the information below first? Or is it very subjective so I shouldn't bother trying to add a heuristic? Or maybe the heuristic is the order is reversed only for arrows?

**Comment by rob-aph** [comment]
Response:
My default for an arrow with information above and below it would be to braille the information below first. I believe the example in GTM 16.5 (with its information above brailled first) is the exception that proves the otherwise consistent rule.

**Comment by NSoiffer** [comment]
I'm not clear on what you mean by 
> I believe the example in GTM 16.5 (with its information above brailled first) is the exception that proves the otherwise consistent rule.

I tried to look for other examples of something above and below, but only found sum and integrals. That is both in the GTM and the APH tutorials website.

What I need to understand is why the GTM example is the way it is so I can capture that example.

**Comment by rob-aph** [comment]
I suspect it is wrong too, but I'm not qualified to assess this.

**Comment by NSoiffer** [comment]
Closing due to lack of more info. Based on what I've heard, the chemistry example is wrong and I've changed the example.
